intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>The text is good but might be hard to understand to a reader with no context.</p><p> </p><p>Perhaps move the example image from below up above these descriptions. So then there's a visual context that a reader can refer back to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I second this suggestion!</p><p> </p>

Reply:
Thanks @dcherian and @JessicaS11 for both of your suggestions on this. I had a hard time with the flow of this notebook and after implementing your comments, I think the structure is much clearer and more logical.

Reply:
<p>So now, it is structured as following. </p><p> </p><ul><li>Examples of indexing on 5x5 2d arrays + the diagram</li><li>Explanation of Pointwise Indexing and Orthogonal Indexing. </li></ul><p> </p><p> </p>